### DISCUSSION TITLE
Adding another device?

### DISCUSSION THREAD

**Discussion by clartan2014** [make-all/tuya-local#888] [General]
Hi! Forgive my ignorance! How do i add another device? Im able to add my 1st device successfully. Now im trying to add the next one, but i dont see the option to add another device? Or do i need to "Add Integration" again? Cheers!

**Comment by make-all** [comment]
I haven't looked closely at the HA 2023.6 and later config UI changes, but prior to that at least, you had to add an integration again.  It may have changed in the new UI  though.
